### PR TITLE
Eth. titles added

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: 3001-4000/LIT3515Epistle.xml [hunk 1]
```diff
@@ -5,6 +5,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <fileDesc>
          <titleStmt>
             <title xml:id="t1">Epistle to the Romans</title>
+            <title xml:id="t2" xml:lang="gez">መልእክተ፡ ጳውሎስ፡ ኀበ፡ ሰብኣ፡ ሮሜ፡</title>
+            <title corresp="#t2" xml:lang="gez" type="normalized">Malʾǝkta Ṗāwlos ḫaba sabʾā Rome</title>
```
Comment (thea-m) [inline 3001-4000/LIT3515Epistle.xml]: sabʾa

File: 3001-4000/LIT3524Epistle.xml [hunk 2]
```diff
@@ -5,6 +5,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <fileDesc>
          <titleStmt>
             <title xml:id="t1">Epistle to the Hebrews</title>
+            <title xml:id="t2" xml:lang="gez">መልእክተ፡ ጳውሎስ፡ ኀበ፡ ሰብኣ፡ ዕብረውየን፡</title>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply 3001-4000/LIT3524Epistle.xml]: was transcribed like this in ESap008, OK

File: 3001-4000/LIT3525Epistle.xml [hunk 3]
```diff
@@ -5,6 +5,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <fileDesc>
          <titleStmt>
             <title xml:id="t1">First Epistle to Timothy</title>
+            <title xml:id="t2" xml:lang="gez">መልእክተ፡ ጳውሎስ፡ ቀዳማዊ፡ ኀበ፡ ሰብኣ፡ ጢሞቴዎስ፡</title>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply 3001-4000/LIT3525Epistle.xml]: in ESap008 ኀበ (no people, correct, copypasted too much, but ኀበ  not la)

File: 3001-4000/LIT3528Epistle.xml [hunk 4]
```diff
@@ -5,6 +5,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <fileDesc>
          <titleStmt>
             <title xml:id="t1">Epistle to Philemon</title>
+            <title xml:id="t2" xml:lang="gez">መልእክተ፡ ጳውሎስ፡ ኀበ፡ ሰብኣ፡ ፊልሞና፡</title>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply 3001-4000/LIT3528Epistle.xml]: as above

**Pull request merged**